### DISCUSSION TITLE
adding plausible.io script in page Head

### DISCUSSION THREAD

**Discussion by artmedia** [oqtane/oqtane.framework#4026] [General] [answered]
Hello,
I'd like to use https://plausible.io/ for my website analytics.
So I need to add this script in page Head of my website:
<script defer data-domain="mywebsite.it" src="https://plausible.io/js/script.js"></script>

I'm trying to add the script as new ResourceType.Script in the Theme project but I don't know how to insert the attributes "defer" and "data-domain".
I'm using v.4.0.6
Thanks in advance

**Comment by artmedia** [comment]
already answered
https://github.com/oqtane/oqtane.framework/discussions/2266

**Comment by leigh-pointer** [comment, marked as answer]
@artmedia add the script in your site settings.
log in navigate to the Site Settings 
![image](https://github.com/oqtane/oqtane.framework/assets/9447612/503c4d31-3d26-4d70-adde-a4b5bc367184)

and add the script there to the **Page Content**
![image](https://github.com/oqtane/oqtane.framework/assets/9447612/5ae88ef1-f263-4a90-bb3e-1fea2c0ed86d)

**Comment by artmedia** [comment]
@leigh-pointer , good solution.
I had seen the property Head content for single page but not for Site.
Great, tnx

**Reply by leigh-pointer** [reply]
@artmedia pleased it helped.  Can you mark this as "Answered" ?
